src/plugins/minimal_scene/MinimalScene.cc

```diff
@@ -1116,6 +1122,26 @@ void MinimalScene::LoadConfig(const tinyxml2::XMLElement *_pluginElem)
       if (!elem->NoChildren())
         ignwarn << "Child elements of <sky> are not supported yet" << std::endl;
     }
+
+    elem = _pluginElem->FirstChildElement("fov");
+    if (nullptr != elem && nullptr != elem->GetText())
+    {
+      double fovDeg;
+      math::Angle fov;
+      std::stringstream fovStr;
+      fovStr << std::string(elem->GetText());
+      fovStr >> fovDeg;
+      if (fovStr.fail())
+      {
+        ignerr << "Unable to set <fov> to '" << fovStr.str()
+               << "' using default fov 90" << std::endl;
```

Comment:
Since the defaults can change, it's better to not hardcode the value in the print statement (authors would have to remember to update the print as well)

```suggestion
               << "' using default horizontal field of view" << std::endl;
```

Reply:
fixed in 7862059